hadoop-ozone/recon/src/test/java/org/apache/hadoop/ozone/recon/fsck/TestContainerHealthTask.java

```diff
@@ -192,7 +195,8 @@ public void testRun() throws Exception {
 
     ReconTaskStatus taskStatus =
         reconTaskStatusDao.findById(containerHealthTask.getTaskName());
-    assertThat(taskStatus.getLastUpdatedTimestamp()).isGreaterThan(currentTime);
+    assertTrue(taskStatus.getLastUpdatedTimestamp() >
+        currentTime);
```

Comment:
Please keep `assertThat`, added by HDDS-10034.

```suggestion
    assertThat(taskStatus.getLastUpdatedTimestamp()).isGreaterThan(currentTime);
```